editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -3916,39 +3925,44 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 	animation_hb->add_child(memnew(VSeparator));
 	animation_hb->hide();
 
-	key_loc_button = memnew(Button("loc"));
-	key_loc_button->set_toggle_mode(true);
-	key_loc_button->set_flat(true);
-	key_loc_button->set_pressed(true);
-	key_loc_button->set_focus_mode(FOCUS_NONE);
-	key_loc_button->add_color_override("font_color", Color(1, 0.6, 0.6));
-	key_loc_button->add_color_override("font_color_pressed", Color(0.6, 1, 0.6));
-	key_loc_button->connect("pressed", this, "_popup_callback", varray(ANIM_INSERT_POS));
-	animation_hb->add_child(key_loc_button);
-	key_rot_button = memnew(Button("rot"));
+	const Color key_accent_color = Color::html("84FFB1");
```

Comment:
this is not needed, the theme does this for you.

Reply:
@djrm Yeah, I figured so much, but I failed to figure out where it is defined or how I'm supposed to retrieve it. Sorry, I'm gonna need some pointers here.

Reply:
@letheed you dont need to set the override manually, it will work because thats defined here https://github.com/godotengine/godot/blob/master/editor/editor_themes.cpp#L484, so just get rid of the color overrides and the colors